src/Cake.Common/Tools/ReportGenerator/ReportGeneratorSettings.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Cake.Core.Annotations;
+using Cake.Core.IO;
+
+namespace Cake.Common.Tools.ReportGenerator
+{
+    /// <summary>
+    /// Contains settings used by <see cref="ReportGeneratorRunner"/>.
+    /// </summary>
+    [CakeAliasCategory("ReportGenerator")]
+    public sealed class ReportGeneratorSettings
```

Comment:
@gep13 probably because of something like this:
```c#
public abstract class Tool<TSettings> where TSettings:BaseSettings
{
...
}
```
which would get rid of all the `toolPath` parameters in the `Run`methods.